target_chains/ethereum/contracts/scripts/receiverSubmitGuardianSetUpgrades.js

```diff
@@ -46,6 +48,7 @@ module.exports = async function (callback) {
         value: 0,
         from: accounts[0],
         gasLimit: 2000000,
+        gasPrice: 200000000000
```

Comment:
Did you want to commit this?

Reply:
no. i missed this my review. removed it.